format_spec/fragment.md

```diff
@@ -22,6 +22,12 @@ my_array                                    # array folder
          |      |_ d1.tdb                   # var-sized dimension (offsets) 
          |      |_ d1_var.tdb               # var-sized dimension (values)
          |      |_ ...      
+         |      |_ t.tdb                    # timestamp attribute
+         |      |_ ...  
+         |      |_ dt.tdb                   # delete timestamp attribute
+         |      |_ ...  
+         |      |_ dcmh.tdb                 # delete condition marker hash attribute
+         |      |_ ...  
```

Comment:
There are filenames here, but the descriptions are barely descriptive. What's the difference between a "timestamp attribute" and a "delete timestamp attribute"? How is a user to know what a "delete condition" is"? What's a "marker hash"?

These are all new files, too, and all their file formats are absent.

Reply:
I added some more information. On the file format, they are considered a fixed attribute so they have the same format as a regular attribute.